package.json

```diff
@@ -54,6 +54,7 @@
     "mocha": "^5.0.0",
     "react": "^16.2.0",
     "react-dom": "^16.2.0",
+    "request": "^2.88.0",
```

Comment:
Don't think this was intentional

Reply:
When I installed it was missing when running tests. I'll double check that its necessary.